server/graphql/general/src/queries/name_property.rs

```diff
@@ -23,12 +24,23 @@ pub fn name_properties(ctx: &Context<'_>) -> Result<NamePropertyResponse> {
 
 pub fn configure_name_properties(
     ctx: &Context<'_>,
+    store_id: &str,
     input: Vec<ConfigureNamePropertyInput>,
 ) -> Result<ConfigureNamePropertiesResponse> {
-    let connection_manager = ctx.get_connection_manager();
+    let user = validate_auth(
+        ctx,
+        &ResourceAccessRequest {
+            resource: Resource::MutateNameProperties,
+            store_id: None,
+        },
+    )?;
+
+    let service_context = ctx
+        .service_provider()
+        .context(store_id.to_string(), user.user_id)?;
```

Comment:
Nit: This can be basic_context since we don't need to validate store id or user_id